apps/betterangels-backend/accounts/models.py

```diff
@@ -115,6 +120,16 @@ class ClientProfile(models.Model):
     veteran_status = TextChoicesField(choices_enum=YesNoPreferNotToSayEnum, blank=True, null=True)
 
 
+class ClientContact(models.Model):
+    client_profile = models.ForeignKey(ClientProfile, on_delete=models.CASCADE, related_name="contacts")
+    name = models.CharField(max_length=100, null=True, blank=True)
+    email = models.EmailField(null=True, blank=True)
+    phone_number = models.CharField(max_length=15, null=True, blank=True)
```

Comment:
Can this be phonenumber field?

https://github.com/BetterAngelsLA/monorepo/blob/d271e7bc47aac526fbf249f156325ce7ca4d6d7e/apps/betterangels-backend/shelters/models.py#L142

Reply:
being worked on in [DEV-631](https://betterangels.atlassian.net/browse/DEV-631), https://github.com/BetterAngelsLA/monorepo/pull/489